packages/dds/sequence/src/test/revertibles.spec.ts

```diff
@@ -274,6 +274,49 @@ describe("Sequence.Revertibles with Remote Edits", () => {
 		collection2 = sharedString2.getIntervalCollection("test");
 	});
 
+	it("interval change, range remove, ack, revert change interval", () => {
+		collection.on("changeInterval", (interval, previousInterval, local, op) => {
+			appendChangeIntervalToRevertibles(
+				sharedString,
+				interval,
+				previousInterval,
+				revertibles,
+			);
+		});
+		sharedString.insertText(0, "hello world");
+		const id = collection.add(0, 5, IntervalType.SlideOnRemove).getIntervalId();
+		containerRuntimeFactory.processAllMessages();
+
+		collection.change(id, 6, 8);
+		sharedString2.removeRange(0, 5);
+		containerRuntimeFactory.processAllMessages();
+
+		revertSharedStringRevertibles(sharedString, revertibles.splice(0));
+		assertIntervals(sharedString, collection, [{ start: 0, end: 0 }]);
+		containerRuntimeFactory.processAllMessages();
+		assertIntervals(sharedString2, collection2, [{ start: 0, end: 0 }]);
+	});
+
+	it("local only text remove, no ack, move interval out of range", () => {
```

Comment:
This test should be moved to a different describe block since there's nothing remote.